client/src/components/DetailView.tsx

```diff
@@ -172,44 +195,56 @@ class DetailViewWithoutI18n extends Component<Props, State> {
                           </div>
-                          <div className="column col-xs-12 col-6">
-                            <b>
-                              <Trans>Business Addresses</Trans>
-                            </b>
-                            <ul>
-                              {detailAddr.businessaddrs &&
-                                detailAddr.businessaddrs.map((rba, idx) => (
-                                  <li key={idx}>{rba}</li>
-                                ))}
-                            </ul>
-                          </div>
-                        </div>
-                        {ownernames && (
                           <div>
-                            <b>
-                              <Trans>People</Trans>
-                            </b>
-                            <ul>
-                              {ownernames.map((owner, idx) => (
-                                <li key={idx}>
-                                  {owner.title.split(/(?=[A-Z])/).join(" ")}: {owner.value}
-                                </li>
-                              ))}
-                            </ul>
+                            {
+                              // Group all contact info by the name of each person/corporate entity
+                              Object.entries(_groupBy(detailAddr.allcontacts, "value"))
+                                // Move head officers and owners to the top of the list
+                                .sort((contact) =>
+                                  contact[1].find(
+                                    (o) => o.title === "HeadOfficer" || o.title.includes("Owner")
+                                  )
+                                    ? -1
+                                    : 0
+                                )
+                                .map((contact, i) => (
+                                  <Accordion title={contact[0]} key={i}>
+                                    {contact[1].map((info, j) => (
+                                      <div className="landlord-contact-info" key={j}>
+                                        <span className="text-bold text-dark">{info.title}</span>
+                                        {info.address && (
+                                          <>
+                                            <br />
+                                            {
+                                              Helpers.formatHpdContactAddress(info.address)
+                                                .addressLine1
+                                            }
+                                            <br />
+                                            {
+                                              Helpers.formatHpdContactAddress(info.address)
+                                                .addressLine2
+                                            }
+                                          </>
+                                        )}
```

Comment:
It might make the code a bit more readable to factor this out into a `<FormattedContactAddress>` component or something but really up to you.  That function could also just bind a variable to `Helpers.formatHpdContactAddress(info.address)` so you wouldn't have to call it twice.